.kitchen.yml

```diff
@@ -0,0 +1,45 @@
+---
+driver:
+  name: vagrant
+  vm_hostname: letsencrypttest.acme.com
+  customize:
+    memory: 1024
+  network:
+    - ["private_network", { ip: "192.168.33.33" }]
+
+provisioner:
+  name: salt_solo
+  formula: letsencrypt
+  pillars:
+    top.sls:
+      base:
+        "*":
+          - letsencrypt
+  state_top:
+    base:
+      "*":
+        - letsencrypt.install
```

Comment:
Would it make sense to run *letsencrypt.config* as well, and add a test to check the ini file content?